src/components/admin/store/PickupEventCard/index.tsx

```diff
@@ -45,6 +51,22 @@ const PickupEventCard = ({ pickupEvent }: PickupEventCardProps) => {
       <Typography variant="h5/regular" suppressHydrationWarning>
         {formatEventDate(start, end, true)}
       </Typography>
+      <Button
+        type="button"
+        className={`${styles.displayButton}`}
+        onClick={() => completePickupEvent(uuid, token)}
+      >
+        <Typography variant="h5/bold">Complete Pickup Event</Typography>
+      </Button>
+      <Button
+        type="button"
+        className={`${styles.displayButton}`}
+        onClick={() => {
+          cancelPickupEvent(uuid, token);
+        }}
+      >
+        <Typography variant="h5/bold">Cancel Pickup Event</Typography>
+      </Button>
```

Comment:
I think you should move these buttons to the pickup event page, next to "Edit Pickup Event"

![image](https://github.com/acmucsd/membership-portal-ui-v2/assets/22133785/1ccd8f97-7a85-42f0-bdd2-90152b923c00)

because it's probably a bad idea to have buttons inside links (clicking on the button also clicks the link)